src/main.rs

```diff
@@ -1,27 +1,52 @@
+use std::{net::IpAddr, str::FromStr};
+
+use clap::Parser;
 use network_time_pester as pest;
 use pest::TestResult;
 
+const DEFAULT_SERVER: &str = "::1:123";
```

Comment:
Why does this include the port when we parse this into an ip addr? Also the syntax isnt entirely right for adding a port to ipv6 addr.